nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -128,18 +128,22 @@ let
     }
   ''));
 
+  mimeTypesConfig = toString (flip mapAttrsToList cfg.mimeTypes (mimeTypes: extension: ''
+    ${mimeTypes} ${concatStringsSep " " extension};
+  ''));
+
   commonHttpConfig = ''
       # Load mime types.
-      include ${cfg.defaultMimeTypes};
+      default_type application/octet-stream;
```

Comment:
Despite the docs claiming otherwise I've experienced nginx settings to sometimes be order dependent. Does it make sense to move `default_type` here?

Reply:
The current move should have no effect on changing the operation of Nginx.